src/pydata_sphinx_theme/assets/styles/components/_switcher-theme.scss

```diff
@@ -7,8 +7,6 @@
   margin: 0 -0.5rem;
   padding: 0; // We pad the `span` not the container
   color: var(--pst-color-text-muted);
-  border-radius: 0;
```

Comment:
This now gets a rounded focus ring so I decided to remove this border-radius rule.